index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=, initial-scale=1.0">
+    <title>border animation</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="frame">
+        <div class="rotate"></div>
+        <div class="cab">
+            <div class="area">
+                <img src="TikTok-Logo.png" alt="">
```

Comment:
Add value in alt attribute, example
```suggestion
                <img src="TikTok-Logo.png" alt="tik-tok-logo">
```